packages/SwingSet/test/state.test.js

```diff
@@ -125,26 +125,27 @@ test('storage helpers', t => {
   kv.set('bar.5', 'b5');
   kv.set('cab.2', 'c');
 
-  t.deepEqual(Array.from(enumeratePrefixedKeys(kv, 'bar')), [
-    'bar.1',
-    'bar.3',
-    'bar.5',
-  ]);
+  t.deepEqual(
+    Array.from(enumeratePrefixedKeys(kv, 'bar')).map(({ key }) => key),
```

Comment:
`Array.from` already accepts a mapper function.
```suggestion
    Array.from(enumeratePrefixedKeys(kv, 'bar'), ({ key }) => key),
```